### PR TITLE
Added kafka authentification

### LINKED ISSUE #1063: Enable Kafka authentification for kafka and bridge by default
**Is your feature request related to a problem? Please describe.**
The transmission between Kafka broker is currently unencrypted

**Describe the solution you'd like**
The transmissions needs to be encrypted and we need to ensure that only authenticated users can connect

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Scarjit**
# Description

Fixes #1063

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [X] Kowl connects to Kafka
- [ ] All kafka based microservices work

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have checked that my issue complies with the [Contributing guidelines](https://github.com/united-manufacturing-hub/united-manufacturing-hub/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Scarjit** [comment]
https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/1064#issuecomment-1118554956

This is fine

**Comment by Scarjit** [comment]
Superseeded by #1067

File: deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+{{if .Values._000_commonConfig.datasources.barcodereader.enabled}}
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml:2]: these changes are probably not wanted. rebase?

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]
Why are there permission changes in here? Can we take them out?

File: deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml (hunk 1 above)
Comment (Scarjit) [reply deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml:2]: I have no idea, why git is even tracking file permissions, especially on windows.
But tbh dropping the executable bit on those files seems fine to me

**Review by JeremyTheocharis — APPROVED** [review]

**Comment by Scarjit** [comment]
@JeremyTheocharis This pr will fail to automerge, due to Gitguardian.

**Comment by JeremyTheocharis** [comment]
@Scarjit fixed gitguardian stuff, still failing

**Pull request merged**